docs/source/alignment.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>interesting would be this plot comparing to persistence (what you could do now already) or uninitialized (later)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>